src/GSoft.AspNetCore.Authentication.ClientCredentialsGrant/AuthenticationBuilderExtensions.cs

```diff
@@ -0,0 +1,48 @@
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace GSoft.AspNetCore.Authentication.ClientCredentialsGrant;
```

Comment:
Changing the namespace to `Microsoft.Extensions.DependencyInjection` as done in Microsoft's [JwtBearerExtensions.cs](https://github.com/dotnet/aspnetcore/blob/v7.0.0/src/Security/Authentication/JwtBearer/src/JwtBearerExtensions.cs#L9) helps consumers to consume the extensions methods without having to import a namespace like in this screenshot:

![image](https://user-images.githubusercontent.com/14242083/236309508-5499c613-0639-494b-800a-7903d80a8834.png)